bgpd/bgp_btoa.c

```diff
@@ -157,7 +157,7 @@ int main(int argc, char **argv)
 		subtype = stream_getw(s);
 		len = stream_getl(s);
 
-		printf("TIME: %s", ctime_r(&now, tbuf));
+		printf("TIME: %s", time_to_string(now, tbuf));
```

Comment:
the lib api isn't a straight replacement for raw ctime_r(), is it - it does a computation?

Reply:
right, will undo this

Reply:
Rewinded this change.